test/uk/gov/hmrc/plasticpackagingtax/registration/views/liability/LiabilityExpectedWeightViewSpec.scala

```diff
@@ -74,6 +74,12 @@ class LiabilityExpectedWeightViewSpec extends UnitViewSpec with Matchers {
       )
     }
 
+    "display liability weight info" in {
+      view.getElementsByClass("govuk-body").text().text() must include(
```

Comment:
Did you really need the double `text().text()` calls?

Reply:
oop good spot! Im surpirsed that even compiles

Reply:
removed. :)